playground/index.js

```diff
@@ -0,0 +1,33 @@
+import { CloudflareD1Connection, Outerbase, equalsNumber } from '../dist/index.js';
+import express from 'express';
+
+const app = express();
+const port = 4000;
+
+app.get('/', async (req, res) => {
+    const d1 = new CloudflareD1Connection(
+        'API_KEY',
+        'ACCOUNT_ID',
+        'DATABASE_ID'
```

Comment:
if you want to use ENV variables, it's something like `process.env.API_KEY`
https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs

Reply:
For now I'm just leaving this as an open-ended playground for whoever to test in.